### PR TITLE
Feature/55 election choice

### PR DESCRIPTION
**Pull request by wkmeijer**
Added functionality to choose an election, for now three choices are hard-coded. When an election is chosen an Election object is parsed to mainactivity.

### CODE REVIEW

File: app/src/main/java/com/digitalvotingpass/digitalvotingpass/Election.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+ */
+
+public class Election implements Parcelable{
+    public String kind;
+    public String place;
+
+    /**
+     * Creates a election object with two attributes.
+     * @param kind - The kind of election. (e.g. municipal, parlement)
+     * @param place - The location of the electino. (e.g. Amsterdam, Den Haag, Noord-Brabant)
```
Comment (klikooo) [inline app/src/main/java/com/digitalvotingpass/digitalvotingpass/Election.java]: electino = election?

File: app/src/main/java/com/digitalvotingpass/digitalvotingpass/MainActivity.java [hunk 2]
```diff
@@ -28,10 +30,16 @@
...
+        Bundle extras = getIntent().getExtras();
+        election = (Election) extras.get("election");
+
         final MainActivity thisActivity = this;
         setContentView(R.layout.activity_main);
         Toolbar appBar = (Toolbar) findViewById(R.id.app_bar);
         setSupportActionBar(appBar);
+        // set the text of the appbar to the selected election
+        getSupportActionBar().setTitle(election.getKind());
+        getSupportActionBar().setSubtitle(election.getPlace());
```
Comment (klikooo) [inline app/src/main/java/com/digitalvotingpass/digitalvotingpass/MainActivity.java:42]: Is it possible to place the title and subtitle in the center, just like in [here]( https://marvelapp.com/39b062h/screen/28342705)

**Review by klikooo — CHANGES_REQUESTED** [review]
Looks good! But I have two small requests

File: app/src/main/java/com/digitalvotingpass/digitalvotingpass/MainActivity.java (hunk 2 above)
Comment (wkmeijer) [reply app/src/main/java/com/digitalvotingpass/digitalvotingpass/MainActivity.java:42]: It is possible, but we will need to add a custom app bar, which in my opinion is not worth it.

Comment (jonathanraes) [reply app/src/main/java/com/digitalvotingpass/digitalvotingpass/MainActivity.java:42]: good point this needs to be done for all activities, just a line in the layout I think

File: app/src/main/java/com/digitalvotingpass/digitalvotingpass/Election.java (hunk 1 above)
Comment (wkmeijer) [reply app/src/main/java/com/digitalvotingpass/digitalvotingpass/Election.java]: fixed

File: app/src/main/java/com/digitalvotingpass/digitalvotingpass/MainActivity.java (hunk 2 above)
Comment (klikooo) [reply app/src/main/java/com/digitalvotingpass/digitalvotingpass/MainActivity.java:42]: Indeed looks like too much trouble

**Review by klikooo — APPROVED** [review]

**Pull request merged**